fission/spec.go

```diff
@@ -960,31 +960,11 @@ func applyResources(fclient *client.Client, specDir string, fr *FissionResources
 // localArchiveFromSpec creates an archive on the local filesystem from the given spec,
 // and returns its path and checksum.
 func localArchiveFromSpec(specDir string, aus *ArchiveUploadSpec) (*fission.Archive, error) {
-	// get root dir
-	var rootDir string
-	if len(aus.RootDir) == 0 {
```

Comment:
No, we cannot do this. If we didn't add root dir to the file path as an absolute path, then we are not able to create the archive if the spec dir is not in the current path.
```
$ fission spec apply --specdir $GOPATH/src/github.com/fission/fission-bank-sample/specs
Fatal error: Failed to apply specs: Archive 'bank-go-zip' is empty
```